flows/update_neo4j.py

```diff
@@ -457,40 +510,320 @@ async def update_concepts(*, dry_run: bool = False) -> None:
+                )
+                aggregated_inference_content = (
+                    aggregated_inference_object["Body"].read().decode("utf-8")
+                )
+                aggregated_inference_json = json.loads(aggregated_inference_content)
+
+                # Skip if the JSON is empty
+                if not aggregated_inference_json:
+                    logger.warning(f"Skipping empty inference file: {document_id_key}")
+                    continue
+            except Exception as e:
+                logger.error(f"Error reading inference file {document_id_key}: {e}")
+                continue
+
+            try:
+                document_data_object = s3.get_object(
+                    Bucket=bucket_name, Key=f"embeddings_input/{filename}"
+                )
+                document_data_string = (
+                    document_data_object["Body"].read().decode("utf-8")
+                )
+                document_data_json = json.loads(document_data_string)
+            except ClientError as e:
+                error_code = e.response.get("Error", {}).get("Code", "")
+                if error_code == "NoSuchKey":
+                    logger.warning(f"Document data not found for {filename}, skipping")
+                    continue
+                else:
+                    raise
+            except Exception as e:
+                logger.error(f"Error reading document data for {filename}: {e}")
+                continue
+
+            # Parse the publication timestamp - use .get() for safety
+            document_metadata = document_data_json.get("document_metadata", {})
+            publication_ts_str = document_metadata.get("publication_ts")
+            if publication_ts_str and publication_ts_str.endswith("Z"):
+                publication_ts_str = publication_ts_str.replace("Z", "+00:00")
+            publication_ts = (
+                datetime.fromisoformat(publication_ts_str)
+                if publication_ts_str
+                else None
+            )
+
+            # Add document data to batch - use .get() for safety
+            document_id = document_data_json.get("document_id")
+            if not document_id:
+                logger.warning("Document missing document_id, skipping")
+                continue
+
+            document_batch.append(
+                {
+                    "document_id": document_id,
+                    "title": document_metadata.get("document_title", ""),
+                    "document_slug": document_metadata.get("slug", ""),
+                    "family_id": document_metadata.get("family_import_id"),
+                    "family_slug": document_metadata.get("family_slug", ""),
+                    "publication_ts": publication_ts.isoformat()
+                    if publication_ts
+                    else None,
+                    "translated": document_data_json.get("translated", False),
+                    "geography_ids": document_metadata.get("geographies", []),
```

Comment:
suggestion: keep `"geographies"` as the name to align with the field names we use elsewhere. I think an extra service that decides to rename some fields will likely lead to confusion. unless you've seen geography ISO codes start being referred to as IDs elsewhere, in which case this makes sense!